download-and-preprocessing.kts

```diff
@@ -46,10 +55,9 @@ fun download() {
         }
     }
 
-    println("\rfiles download success: ${saveFile.absolutePath}")
+    println("\rfiles download success, save in ${targetFile.absolutePath}")
```

Comment:
Could we import and use Ktor here instead (just for code clarity), or is it too overkill 😅